src/extension.ts

```diff
@@ -1197,7 +1197,9 @@ function parseName(line: string): string {
 async function getContainers(resource: ContainerContainer): Promise<Container[] | undefined> {
     const q = shell.isWindows() ? `'` : `"`;
     const lit = (l: string) => `{${q}${l}${q}}`;
-    const query = `${lit("NAME\\tIMAGE\\n")}{range ${resource.containersQueryPath}.containers[*]}{.name}${lit("\\t")}{.image}${lit("\\n")}{end}`;
+    const query = `${lit("NAME\\tIMAGE\\n")}{range ${resource.containersQueryPath}.initContainers[*]}{.name}\
```

Comment:
Cool, yes, I like that better.

Reply:
I really do like this idea but it turns on that is a pretty significant change if I am understanding the structure and flow correctly.  The "(init)" suffix causes the (init)container not to be found when the command (logs, for example) runs and therefore it fails.  That structure has the name and the image.  I assume it could be modified to add a type but I have not looked that far.  Personally, I tend to know which ones are the initContainers because of the names.

Reply:
@lstocchi  I went ahead and added the property to the Container.